libavcodec/jpeg2000dec.c

```diff
@@ -479,6 +479,24 @@ static int get_siz(Jpeg2000DecoderContext *s)
     return 0;
 }
 
+static int get_cap(Jpeg2000DecoderContext *s, int len)
+{
+    av_log(s->avctx, AV_LOG_ERROR, "Parsing of CAP segment not implemented\n");
```

Comment:
```suggestion
    av_log(s->avctx, AV_LOG_INFO, "Parsing of CAP segment not implemented\n");
```

I am not sure it rises to the level of error.